awscli/data/cli.json

```diff
@@ -66,6 +66,13 @@
             "dest": "connect_timeout",
             "type": "int",
             "help": "<p>The maximum socket connect time in seconds. If the value is set to 0, the socket connect will be blocking and not timeout.</p>"
+        },
+        "cli-binary-format": {
+            "choices": [
+                "base64",
+                "legacy"
+            ],
+            "help": "<p>The formatting style to be used for binary blobs. If the format is set to legacy, the value passed to the AWS CLI for a blob field will be treated literally for input and binary blobs will be represented as a Base64 encoded string in output. The legacy format is consistent with V1 of the AWS CLI. The base64 format will treat the values passed to the AWS CLI for a blob field as a Base64 encoded string, decoding it into the appropriate binary value for input, and binary blobs will be represented as a Base64 encoded string in output.</p>"
```

Comment:
Small nitpicks:

* It may be useful to describe the default first here as most people will probably using that.
* It also may be worth to note what the default is.
* In general, it may be worth not including the explanation on how the CLI does serialization and just keep it as if base64, blobs must be base64 encoded and if legacy blobs must not be base64 encoded. This will make the description and a little more succinct.

Reply:
Updated